src/navigation/wallet/screens/KeyOverview.tsx

```diff
@@ -230,6 +239,7 @@ const KeyOverview: React.FC<KeyOverviewScreenProps> = ({navigation, route}) => {
   const defaultAltCurrency = useAppSelector(({APP}) => APP.defaultAltCurrency);
   const [showKeyDropdown, setShowKeyDropdown] = useState(false);
   const key = keys[id];
+  const {hideKeyBalance} = key;
```

Comment:
This will throw an err while deleting the key. Can be fixed in couple of ways
1. `const {hideKeyBalance} = key || {};`
2. on line no: 422 `!key?.hideKeyBalance` instead `!hideKeyBalance`